Sources/RSDKUtilsMain/Extensions/Bundle+EnvironmentInformation.swift

```diff
@@ -11,11 +14,15 @@ import UIKit
 
 extension Bundle: BundleProtocol {
     public var valueNotFound: String {
-        return ""
+        ""
     }
 
     public func osVersion() -> String {
-        return UIDevice.current.systemVersion
+        #if os(watchOS)
+        WKInterfaceDevice.current().systemVersion
```

Comment:
Interesting. Do we have any RAT tracking from watchOS apps? @donnie-jp

Reply:
Not sure... maybe 🤔